Services/DataX.Config/DataX.Config.Local.Test/Resource/flowSaved.json

```diff
@@ -294,6 +294,7 @@
   "gui": {
     "name": "localconfiggentest",
     "displayName": "localconfiggentest",
+    "databricksToken": null,
```

Comment:
can we also have a test that doesn't have these defined?

Reply:
going forward save api will always add this. based on sparktype this will either have a value or be null